code/modules/spells/spell_types/conjure/simian.dm

```diff
@@ -0,0 +1,111 @@
+/datum/action/cooldown/spell/conjure/simian
+	name = "Summon Simians"
+	desc = "This spell reaches deep into the elemental plane of bananas (the monkey one, not the clown one), and \
+		summons monkeys and gorillas that will promptly flip out and attack everything in sight. Fun! \
+		Their lesser, easily manipulable minds will be convinced you are one of their allies, but only for a minute. Unless you also are a monkey."
+	button_icon_state = "bee"
+	sound = 'sound/ambience/antag/monkey.ogg'
+
+	school = SCHOOL_CONJURATION
+	cooldown_time = 2 MINUTES
+	cooldown_reduction_per_rank = 15 SECONDS
+
+	invocation = "OOGA OOGA OOGA!!!!"
+	invocation_type = INVOCATION_SHOUT
+
+	summon_radius = 2
+	summon_type = list(/mob/living/carbon/human/species/monkey/angry, /mob/living/carbon/human/species/monkey/angry, /mob/living/simple_animal/hostile/gorilla/lesser)
+	summon_amount = 4
+
+/datum/action/cooldown/spell/conjure/simian/level_spell(bypass_cap)
+	. = ..()
+	summon_amount++ // MORE, MOOOOORE
+	if(spell_level == spell_max_level) // We reward the faithful.
+		summon_type = list(/mob/living/carbon/human/species/monkey/angry, /mob/living/simple_animal/hostile/gorilla)
+
+/datum/action/cooldown/spell/conjure/simian/cast(atom/cast_on)
+	. = ..()
+	if(FACTION_MONKEY in owner.faction)
```

Comment:
Prefer to use `cast_on` than `owner`, they're the same for conjure spells but it allows for future coders to cast your spell without an owner